server/src/domain/asset/asset.service.ts

```diff
@@ -187,8 +187,19 @@ export class AssetService {
     return folder;
   }
 
-  getMapMarkers(auth: AuthDto, options: MapMarkerDto): Promise<MapMarkerResponseDto[]> {
-    return this.assetRepository.getMapMarkers(auth.user.id, options);
+  async getMapMarkers(auth: AuthDto, options: MapMarkerDto): Promise<MapMarkerResponseDto[]> {
+    const userIds: string[] = [auth.user.id];
+    if (1) {
```

Comment:
Is the idea to show partner assets under some condition? Like if the url contains `withPartner=true` or something?

Reply:
for the `if(1)` you mean?   Yes,  I was thinking about it.. But kind of decided against it..  open for input! :)
I think it's "obvious" that you want the partners photos to behave like your own if `partner.inTimeline` is set.  The downside is that we'll always to the more complex DB-lookup.


will remove the if(1) or add the withPartner condition, depending on the input

Reply:
Sure I can. Remove the dependency on `partner.inTimeline` then?

Arguably both convey the desire to include partner's photos in the response, and it's kind of illogical to require both.

Reply:
I don't think we need to keep it or use it to filter the partner list, no.